python/src/scipp/plot/controller3d.py

```diff
@@ -59,21 +65,48 @@ def _get_axes_parameters(self):
         """
         axparams = {}
         if self.positions is not None:
-            axparams = self.pos_axparams
+            extents = self.model.get_positions_extents()
+            axparams = {
+                xyz: {
+                    "lims": ex["lims"],
+                    "label": name_with_unit(1.0 * ex["unit"], name=xyz.upper())
+                }
+                for xyz, ex in extents.items()
+            }
         else:
             axparams = super()._get_axes_parameters()
 
+        axparams["box_size"] = np.array([
+            axparams['x']["lims"][1] - axparams['x']["lims"][0],
+            axparams['y']["lims"][1] - axparams['y']["lims"][0],
+            axparams['z']["lims"][1] - axparams['z']["lims"][0]
+        ])
+
+        for i, xyz in enumerate("xyz"):
+            axparams[xyz]["scaling"] = 1.0 / axparams["box_size"][
+                i] if self.aspect == "auto" else 1.0
+            axparams[xyz]["lims"] *= axparams[xyz]["scaling"]
+
+        axparams["box_size"] *= np.array([
+            axparams['x']["scaling"], axparams['y']["scaling"],
+            axparams['z']["scaling"]
+        ])
+
         axparams["centre"] = [
             0.5 * np.sum(axparams['x']["lims"]),
             0.5 * np.sum(axparams['y']["lims"]),
             0.5 * np.sum(axparams['z']["lims"])
         ]
 
-        axparams["box_size"] = np.array([
-            axparams['x']["lims"][1] - axparams['x']["lims"][0],
-            axparams['y']["lims"][1] - axparams['y']["lims"][0],
-            axparams['z']["lims"][1] - axparams['z']["lims"][0]
-        ])
+        if self.pixel_size is not None:
+            axparams["pixel_size"] = self.pixel_size
+        else:
+            if self.positions is not None:
+                axparams["pixel_size"] = 0.05 * np.amin(axparams["box_size"])
```

Comment:
What is 0.05 here?

Reply:
some arbitrary number, just like what we use to scale the tick labels: https://github.com/scipp/scipp/blob/master/python/src/scipp/plot/figure3d.py#L258

give it a go and maybe we can find a better default value?

Reply:
Note that we could make the `pixel_size` part of the information of the instrument when it is loaded (i.e. scipp would know the pixel_size for Loki or Dream), so that this value of 0.05 is in practice rarely used.

Reply:
I don't think this is the right approach, it would pollute the code or the data with such params.

I have taken a similar but different approach for LoKI, defining `loki.instrument_view()` in the `loki` helper module, which can then also include desired other default settings such as camera position.

Reply:
Yes, this is also a good option.